src/mlpack/tests/split_data_test.cpp

```diff
@@ -121,6 +121,29 @@ TEST_CASE("SplitDataResultMat", "[SplitDataTest]")
   CheckMatrices(input, concat);
 }
 
+TEST_CASE("SplitDataResultField", "[SplitDataTest]")
+{
+  field<mat> input(1, 2);
+
+  mat matA(2, 10);
+  mat matB(2, 10);
+
+  size_t count = 0; // Counter for unique sequential values.
+  matA.imbue([&count]() { return ++count; });
+  matB.imbue([&count]() { return ++count; });
+
+  input(0, 0) = matA;
+  input(0, 1) = matB;
+
+  const auto value = Split(input, 0.5, false);
```

Comment:
Do you think we could also add a test for the `Split()` overload for labeled data that takes `arma::field`s?

Reply:
Added in the latest patch.